### PR TITLE
slight changes

### PR DESCRIPTION
**Pull request by kylechi05**

### CODE REVIEW

File: client/src/components/CourseListItem.jsx [hunk 1]
```diff
@@ -30,9 +30,9 @@ const CourseListItem = ({course, navigate}) => {
             {course["SEMESTER"]} {course["YEAR"]}
           </p>
         </div>
-        <p>
+        <p className="items-center flex pl-2">
           <FontAwesomeIcon icon={faUser} className="text-yellow-400 text-xl" />{" "}
-          <span className="text-xl">{classTotal}</span>
+          <span className="text-xl pl-1.5">{classTotal}</span>
```
Comment (kylechi05) [inline client/src/components/CourseListItem.jsx:35]: spacing between icon and class total

File: client/src/components/CourseListItem.jsx
Comment (kylechi05) [inline client/src/components/CourseListItem.jsx:1]: makes icon and class count side-by-side

File: client/src/components/Pagination.jsx
Comment (kylechi05) [inline client/src/components/Pagination.jsx:1]: adds dark mode to page numbers

File: client/src/components/PieGraph.jsx
Comment (kylechi05) [inline client/src/components/PieGraph.jsx:1]: removes axis from pie chart

**Comment by liaozhuzhu** [comment]
Deleted side by side icons since I cleaned it up, besides that everything else looks good. Merging into main now.

**Pull request merged**